### ISSUE TITLE
"Unable to boot device in current state: Booted"  Popup occur while running tests

### ISSUE THREAD

**Issue by divyank31** [appium/appium#16418]
## The problem

Getting a popup "Unable to boot device in current state: Booted" while running iOS 15 tests. Xcode used is 13.0

## Environment

* Appium version that exhibits the issue: 1.22.2
* Last Appium version that did not exhibit the issue (if applicable): 1.22.1
* Desktop OS/version used to run Appium: macOS Monterey
* Node.js version (unless using Appium.app|exe): v17.2.0
* Npm or Yarn package manager: npm
* Mobile platform/version under test: iPhone 13 Pro
* Real device or emulator/simulator: emulator/simulator
* Appium CLI or Appium.app|exe: Appium CLI

## Details

* Checked the device before running the test. It was it Shutdown state. 
* Before starting the test, while booting the device the pop occurred (Unable to boot device in current state: Booted).

<img width="267" alt="Screenshot 2022-02-07 at 7 50 59 PM" src="https://user-images.githubusercontent.com/68773301/152805723-aebd68de-8fa5-45d2-b165-7b902ef3018e.png">

* This pop prevents the mobile device to open. 
* If we close the pop-up, the Mobile screen got visible.

## Link to Appium logs

https://gist.github.com/divyank31/7c15bd7cbba000b1d7d9b58be121e773

## Code To Reproduce Issue [ Good To Have ]

Please remember that with sample code it's easier to reproduce the bug and it's much faster to fix it.

**Comment by divyank31** [comment]
@mykola-mokhnach I don't think it's an Apple bug as the same is working fine with Appium version 1.22.1 .  
Though this popup has no influence on the test progress but I am not able to record a video of the test progress as this popup blocks the devices from being visible on the screen.

**Comment by mykola-mokhnach** [comment]
Please provide both logs when this issue happens and when it does not

**Comment by gauthierm** [comment]
I can see the same dialog appear if I perform the following:

- Run tests
- Simulator stays running
- Quit simulator with Cmd+Q
- Run tests again
- Simulator starts
- Boot error message appears

I am also using iOS 15 on the iPhone 13 Pro simulator.

I've tried Appium versions 1.22.0, 1.22.1, and 1.22.2 (installed via NPM) and got the same dialog box with the reproduce steps.

If I leave the simulator running after tests are finished, subsequent runs do not cause the boot message to appear.

**Comment by gauthierm** [comment]
I think my issue is caused by this setting (which is on by default):

![Screen Shot 2022-02-08 at 17 16 58](https://user-images.githubusercontent.com/120511/153077336-e15255cf-d084-4d3d-8adf-c43ad2db9153.png)

For CI machines this can be configured, but ideally Appium could cleanly handle the case when this setting is enabled for developer machines.

**Comment by mykola-mokhnach** [comment]
It should not be too hard to disable the simulator setting above. Let me try it.

**Comment by mykola-mokhnach** [comment]
@gauthierm Please check if https://github.com/appium/appium-ios-simulator/pull/320 resolves the case

**Comment by mykola-mokhnach** [comment]
The patch has been published. It could be tested in Appium2. Since the xcuitest driver has not been published yet with this module included it must be installed from github source as described in https://appiumpro.com/editions/122-installing-appium-20-and-the-driver-and-plugins-cli

**Comment by divyank31** [comment]
@mykola-mokhnach  Can we get this fix in Appium1 as well?

**Comment by mykola-mokhnach** [comment]
Unfortunately we cannot as we stopped maintaining Appium 1 since the new year. All the current team effort is directed to improve Appium2

**Comment by Akash-nykaa** [comment]
But this is happening with appium 2

**Comment by jlipps** [comment]
@Akash-nykaa as this issue has been closed as completed, please open a new issue with all the relevant information if you'd like to report a bug related to the latest version of the appium 2 driver.
